From 17c522606cbada2da27a00e0e94bd8a4b5e374ba Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Wed, 6 Sep 2017 19:30:01 +0800
Subject: [PATCH 3309/5242] MLK-17491-5 dts: imx7ulp-evk: remove unneccesary
 bootargs in chosen node

commit  5b8ae926ef3fb5d9dcc6be6c8ada6fb8f169fe54 from
https://source.codeaurora.org/external/imx/linux-imx.git

Earlycon can be simply turned on by adding "earlycon" in bootargs,
then of earlycon core will automatically find the matching earlycon device
via stdout-path during early boot. No need specify address and baudrate
which is hard to use.

Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 1bbd06a..8483067 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -22,7 +22,6 @@
 	};
 
 	chosen {
-		bootargs = "console=ttyLP0,115200 earlycon=lpuart32,0x402D0000,115200";
 		stdout-path = &lpuart4;
 	};
 
-- 
1.7.9.5

